Supplementary information for Altermatt et al. Methods in Ecology and Evolution. DOI: 10.1111/2041-210X.12312

**‚ÄúBig answers from small worlds: a user's guide for protist microcosms as a model system in ecology and evolution‚Äù **

Altermatt F, Fronhofer EA, Garnier A, Giometto A, Hammes F, Klecka J, Legrand D, M√§chler E, Massie TM, Pennekamp F, Plebani M, Pontarp M, Schtickzelle N, Thuillier V & Petchey OL

**2.7 DNA Sequencing and Barcoding**

**Introduction**

DNA sequencing of protist species is done to analyse population dynamics ([Hajibabaei *et al.* 2011](#_ENREF_14); [Zufall, Dimon & Doerder 2013](#_ENREF_29)) or genetic diversity of species complex ([e.g., Catania *et al.* 2009](#_ENREF_6)), for comparative studies ([Gray *et al.* 1998](#_ENREF_13)) or to understand the evolution of gene and genomes ([Brunk *et al.* 2003](#_ENREF_5); [Chen, Zhong & Monteiro 2006](#_ENREF_8); [Moradian *et al.* 2007](#_ENREF_21)). DNA barcoding is a special case of sequencing, focusing on a short and conserved portion of the genome in order to disentangle the phylogenetic relationships between taxa (Pawlowski *et al*. 2012). The use of DNA barcoding or sequencing enables to estimate nucleotide diversity and fixation indices (Fst), consequently to access the genetic structure and gene flow within and among populations. The genetic variability can also be compared to life history traits or phenotypic plasticity resulting from local adaptation ([Krenek, Petzoldt & Berendonk 2012](#_ENREF_18)) in order to understand the pattern of evolution. DNA barcoding has been of great interest in phylogenetics to discover morphospecies or cryptic species and to identify the species‚Äô composition in a particular environment. Barcodes have been used to study the composition and interaction between species coming from the same environment, like soil ([Blaxter 2004](#_ENREF_4)) or water column ([Stern *et al.* 2010](#_ENREF_26); [Hajibabaei *et al.* 2011](#_ENREF_14)) and identify cryptic or morpho-species frequent in protists ([e.g., Barth *et al.* 2006](#_ENREF_3)).

The choice of the gene or barcode of interest should be carefully made depending on the taxonomic level and species one works on. Knowing that the mitochondrial genome evolves faster, the accumulation of sequence variability between organisms would be higher, enabling to discriminate the intraspecific relationships or recent phylogenetic splits. Many barcodes have been previously developed ([Nassonova *et al.* 2010](#_ENREF_22); [Pawlowski *et al.* 2012](#_ENREF_23)) either on the mitochondrial genome (e.g., Cox-1, Cob, SSU of rDNA) or in the nuclear genome (e.g., ITS1-2, SSU and LSU of rDNA).

*Approaches*

The procedure for DNA sequencing and barcoding consists of three steps: DNA extraction, gene/barcode choice and amplification by PCR, and sequencing methods.

First, DNA should be extracted from the cell. Different procedures have been developed to isolate the whole DNA: Chelex solution ([Walsh, Metzger & Higuchi 1991](#_ENREF_28)), various DNA isolation kits (e.g., Epicentre) or modified phenol/chloroform extraction ([Couvillion & Collins 2012](#_ENREF_9)). The extraction of whole DNA is sufficient for known barcodes or single copy gene sequencing. However, many protist species are polyploid (\>45 in *Tetrahymena thermophila*), and some events of duplication of particular genes well known. Furthermore, ribosomal genes have been duplicated from the mitochondrial genome to the nuclear genome. In that particular case, all copies will be amplified without distinction. Since these two genomes do not evolve at the same speed, a mixture of amplified sequences will increase the noise on the chromatogram. This will complicate the readability of the resulting sequence and can lead to false interpretations. When one wants to create new barcodes and ensure their specificity, the mitochondrial genome should be separated from the nuclear genome as a necessary precaution. The separation between nuclear and mitochondrial materials could be achieved by migration on agarose gel (0.4% at 50V for 6h) with total DNA isolated by modified chloroform extraction (V. Thuillier et al. unpub. results). Depending on the organism being studied, the upper and brighter band in the agarose gel corresponds to the nuclear DNA and the mitochondrial DNA appears around 40kb. The band of interest could be excised and purified by a kit (e.g., wizard SV kit). In ciliates, two nuclei are found: the macronucleus participates in the somatic division and the micronucleus, which is responsible for the germinal line. Both genomes are particularly similar given that the micronucleus genes are copied several times to form the macronucleus ([Prescott 1994](#_ENREF_24)). Therefore, in order to analyse nuclear genes, the two nuclei should be separated by gradient separations, such as Percoll gradients ([Allen 1999](#_ENREF_1); [Asai & Forney 2000](#_ENREF_2)).

Second, the choice of the gene of interest or barcode should be carefully made depending on the taxonomic level and species one works on. Knowing that the mitochondrial genome evolves faster, the accumulation of sequence variability between organisms would be higher, enabling to discriminate the intraspecific relationships or recent phylogenetic node. Many barcodes have be developed ([Nassonova *et al.* 2010](#_ENREF_22); [Pawlowski *et al.* 2012](#_ENREF_23)) either on the mitochondrial genome (e.g., Cox-1 cytochrome oxydase 1, in *Tetrahymena*, in Amoebae Cob cytochrome b, SSU of rDNA ribosomal small sub-unit, [Slapeta, Moreira & Lopez-Garcia 2005](#_ENREF_25); [Chantangsi *et al.* 2007](#_ENREF_7); [Nassonova *et al.* 2010](#_ENREF_22); [Kher *et al.* 2011](#_ENREF_17)) or in fast evolving nuclear portions (e.g. ITS1-2 internal transcribed spacer 1-2 in *Carchesium* *polypinum*, diatoms, and *Tetrahymena* *thermophila,* SSU rDNA 5.8S in *Paramecium* *aurelia* or LSU rDNA ribosomal large sub-unit, [Chen, Zhong & Monteiro 2006](#_ENREF_8); [Catania *et al.* 2009](#_ENREF_6); [Gentekaki & Lynn 2009](#_ENREF_12); [Moniz & Kaczmarska 2010](#_ENREF_20)). The PCR conditions and primers used are described in the corresponding publications. New barcodes could also be designed with Primer3 software (<http://bioinfo.ut.ee/primer3-0.4.0/>) that helps to design primers in association with NCBI database. A classical procedure for the PCR ([Chen, Zhong & Monteiro 2006](#_ENREF_8)) could be tested and modified if necessary knowing that the Tm (melting point temperature) has a strong influence on the PCR functioning. An optimal PCR protocol can be achieved by testing across a temperature-magnesium gradient.

Finally, the PCR products could be sequenced by Sanger Sequencing method or Next Generation Sequencing (NGS) ([llumina, Solexa, Solid, see Valentini, Pompanon & Taberlet 2009](#_ENREF_27)). The use of Sanger method is favoured when the number of sequences and barcodes are limited. NGS costs have much decreased recently. NGS are usually used in metagenomics or in comparative studies. For Sanger methods, resulting sequences should be cleaned, most often achieved visually on the chromatogram in MEGA (open source software, http://www.megasoftware.net/), Sequencher (open source software, http://www.genecodes.com/download/external-tools-download) or Geneious (private software, http://www.geneious.com/download). For the analysis of the sequences, many software exist and are well explained ([Hall 2013](#_ENREF_16)) depending on the purpose. The treatment and analysis of the sequences generated requires expertise in bioinformatics and the detailed procedure is out of scope of this paper. NGS are usually used in metagenomics ([Hajibabaei *et al.* 2011](#_ENREF_14)), surveys of microorganism diversity ([Medinger *et al.* 2010](#_ENREF_19)) or in comparative studies. Sequencing data are available and compiled into various databases like GenBank (NCBI, <http://www.ncbi.nlm.nih.gov/>) and BOLD (Barcode of Life Data Systems, <http://www.barcodinglife.org>*/*) for the barcoding sequences.

**Materials**

***Equipment***

A standard molecular biology laboratory (including a fume hood) and respective equipment is needed.

***Reagents***

*DNA extractions (Phenole/Chloroform extraction),* Lysis buffer (pH 9.5):

-   10mM Tris, pH 7.5

-   0.5M EDTA

-   1% SDS, completed with ultrapure water

*DNA extractions (modified Chloroform extraction, modified by V. Thuillier et al.),* Lysis buffer (pH=8):

-   Tris 20 mM pH 7.5

-   EDTA 1 mM

-   NaCl 100 mM

-   SDS 10%

-   ddH<sub>2</sub>O

*Choice of Barcode and PCR amplification:*

Choice of Cox-1 barcode with ([Chantangsi *et al.* 2007](#_ENREF_7)) forward primer 5‚Äô-ATGTGAGTTGATTTTATAGA-3‚Äô and reverse primer 5‚Äô-CTCTTCTATGTCTTAAACCAGGCA-3‚Äô.

**Procedure**

*DNA extractions (Phenole/Chloroform extraction):*

1.  Collect 2.5\*10<sup>5</sup> cells in 50 *ùúá*LTris (10 mM, pH 7.5).

2.  Re-suspend and add 200 *ùúá*L of pre-heated Lysis buffer (60 ¬∞C).

3.  Add two volumes water and incubate at 60 ¬∞C at least 1 h.

4.  Cool to room temperature, add 50 *ùúá*g/mL proteinase K and incubate at 37 ¬∞C overnight.

5.  Purify with one volume phenol/chloroform/isoamid.

6.  Precipitate with one-tenth volume sodium acetate (pH 5.2) and one volume of isopropanol.

7.  Wash pellet in 70 % ethanol.

8.  Re-suspend in 75*ùúá*L Tris-EDTA.

9.  Add 0.8 *ùúá*g/*ùúá*L RNase A and incubate for 30min at 37 ¬∞C.

10. Purify with one volume phenol/chloroform/isoamid.

11. Precipitate with one-tenth volume sodium acetate (pH 5.2) and one volume of isopropanol.

12. Wash pellet in 70% ethanol.

13. Re-suspend in the desired volume of Tris-EDTA.

*DNA extractions (modified Chloroform extraction, modified by V. Thuillier et al.):*

1.  Dilute the amount of cells in order to have a final volume of 200 ¬µl ultrapure H<sub>2</sub>O.

2.  Add 500 ¬µL of lysis Buffer (pH=8) and vortex them for few seconds until all is homogenized. Then, add 10 ¬µL proteinase k (mg/¬µl).

3.  Inverse the tube 2-3 times.

4.  Incubate at 37 ¬∞C for 20 min, then vortex for few seconds.

5.  Inactivate the enzyme by incubation for 20 min at 65 ¬∞C.

6.  Add 10mg/mL RNAse A, mix gently and incubate for 30 min at 37 ¬∞C. Vortex for few seconds.

7.  Separation with 750 ¬µl of chloroform-isoamid (24:1). Then, homogenise and centrifuge at 12¬†000 rcf for 10min at room temperature. Collect the upper phase (aqueous phase).

8.  Separation with <span id="OLE_LINK1" class="anchor"></span>750 ¬µL chloroform-isoamid (24:1) and repeat the same process.

9.  Precipitation with 1 mL ethanol 100% (-20 ¬∞C) at room temperature. Mix carefully and incubate for 15 mins.

10. Centrifugation at 10000 rpm for 30mins and return the tube to eliminate the ethanol.

11. Washing with 1 mL ethanol 70% and centrifuge for 5 min at 8000 rpm. Remove the ethanol with a pipette. Dry only if it rests some ethanol for few minutes.

12. Dissolution in 20 ¬µL of water.

*Choice of Barcode and PCR amplification (Chantangsi et al. 2007 for COX-1):*

1.  Initial denaturation step of 94 ¬∞C for 4 min.

2.  Followed by 5 cycles consisting of (each cycle): 30 s at 94 ¬∞C; 1 min at 45¬†¬∞C; 105 s at 72 ¬∞C.

3.  Followed by 35 cycles consisting of (each cycle): 30 s at 94 ¬∞C; 1 min at¬†55 ¬∞C; s at 72 ¬∞C.

4.  Final extension step at 72¬∞C for 10 min.

Classical procedure for the PCR ([Chen, Zhong & Monteiro 2006](#_ENREF_8)):

1.  Initial denaturation step of 94¬∞C for 10 min.

2.  Followed by 30 cycles consisting of (each cycle): 1 min at 94 ¬∞C, 1 min at¬†Tm; 1 min at 72 ¬∞C.

3.  Final extension step at 72 ¬∞C for 10 min.

**References**

<span id="_ENREF_1" class="anchor"></span>Allen, S.L. (1999) Chapter 8 Isolation of Micronuclear and Macronuclear DNA. *Methods in Cell Biology* (eds J.A. David & D.F. James), pp. 241-252. Academic Press.

<span id="_ENREF_2" class="anchor"></span>Asai, D.L. & Forney, J.D. (2000) *Tetrahymena termophila*. Academic Press, San Diego.

<span id="_ENREF_3" class="anchor"></span>Barth, D., Krenek, S., Fokin, S.I. & Berendonk, T.U. (2006) Intraspecific genetic variation in Paramecium revealed by mitochondrial cytochrome C oxidase I sequences. *J Eukaryot Microbiol,* **53,** 20-25.

<span id="_ENREF_4" class="anchor"></span>Blaxter, M.L. (2004) The promise of a DNA taxonomy. *Philos Trans R Soc Lond B Biol Sci,* **359,** 669-679.

<span id="_ENREF_5" class="anchor"></span>Brunk, C.F., Lee, L.C., Tran, A.B. & Li, J. (2003) omplete sequence of the mitochondrial genome of *Tetrahymena thermophila* and comparative methods for identifying highly divergent genes. *Nucleic Acids Research,* **31,** 1673-1682.

<span id="_ENREF_6" class="anchor"></span>Catania, F., Wurmser, F., Potekhin, A.A., Przybo≈õ, E. & Lynch, M. (2009) Genetic Diversity in the Paramecium aurelia Species Complex. *Molecular Biology and Evolution,* **26,** 421-431.

<span id="_ENREF_7" class="anchor"></span>Chantangsi, C., Lynn, D.H., Brandl, M.T., Cole, J.C., Hetrick, N. & Ikonomi, P. (2007) Barcoding ciliates: a comprehensive study of 75 isolates of the genus Tetrahymena. *Int J Syst Evol Microbiol,* **57,** 2412-2425.

<span id="_ENREF_8" class="anchor"></span>Chen, B., Zhong, D. & Monteiro, A. (2006) Comparative genomics and evolution of the HSP90 family of genes across all kingdoms of organisms. *BMC Genomics,* **7,** 156.

<span id="_ENREF_9" class="anchor"></span>Couvillion, M.T. & Collins, K. (2012) Chapter 12 - Biochemical Approaches Including the Design and Use of Strains Expressing Epitope-Tagged Proteins. *Methods in Cell Biology* (ed. C. Kathleen), pp. 347-355. Academic Press.

<span id="_ENREF_10" class="anchor"></span>Diggles, B.K. & Adlard, R.D. (1997) Intraspecific variation in Cryptocaryon irritans. *J Eukaryot Microbiol,* **44,** 25-32.

<span id="_ENREF_11" class="anchor"></span>Foissner, W., Chao, A. & Katz, L.A. (2007) Diversity and geographic distribution of ciliates (Protista: Ciliophora). *Biodiversity and Conservation,* **17,** 345-363.

<span id="_ENREF_12" class="anchor"></span>Gentekaki, E. & Lynn, D.H. (2009) High-level genetic diversity but no population structure inferred from nuclear and mitochondrial markers of the peritrichous ciliate Carchesium polypinum in the Grand River basin (North America). *Appl Environ Microbiol,* **75,** 3187-3195.

<span id="_ENREF_13" class="anchor"></span>Gray, M.W., Lang, B.F., Cedergren, R., Golding, G.B., Lemieux, C., Sankoff, D., Turmel, M., Brossard, N., Delage, E., Littlejohn, T.G., Plante, I., Rioux, P., Saint-Louis, D., Zhu, Y. & Burger, G. (1998) Genome structure and gene content in protist mitochondrial DNAs. *Nucleic Acids Research,* **26,** 865-878.

<span id="_ENREF_14" class="anchor"></span>Hajibabaei, M., Shokralla, S., Zhou, X., Singer, G.A.C. & Baird, D.J. (2011) Environmental Barcoding: A Next-Generation Sequencing Approach for Biomonitoring Applications Using River Benthos. *PLoS ONE,* **6,** e17497.

<span id="_ENREF_15" class="anchor"></span>Hajibabaei, M., Singer, G.A., Clare, E.L. & Hebert, P.D. (2007) Design and applicability of DNA arrays and DNA barcodes in biodiversity monitoring. *BMC Biol,* **5,** 24.

<span id="_ENREF_16" class="anchor"></span>Hall, B.G. (2013) Building phylogenetic trees from molecular data with MEGA. *Mol Biol Evol,* **30,** 1229-1235.

<span id="_ENREF_17" class="anchor"></span>Kher, C.P., Doerder, F.P., Cooper, J., Ikonomi, P., Achilles-Day, U., Kupper, F.C. & Lynn, D.H. (2011) Barcoding Tetrahymena: discriminating species and identifying unknowns using the cytochrome c oxidase subunit I (cox-1) barcode. *Protist,* **162,** 2-13.

<span id="_ENREF_18" class="anchor"></span>Krenek, S., Petzoldt, T. & Berendonk, T.U. (2012) Coping with temperature at the warm edge--patterns of thermal adaptation in the microbial eukaryote Paramecium caudatum. *PLoS ONE,* **7,** e30598.

<span id="_ENREF_19" class="anchor"></span>Medinger, R., Nolte, V., Pandey, R.V., Jost, S., Ottenwalder, B., Schlotterer, C. & Boenigk, J. (2010) Diversity in a hidden world: potential and limitation of next-generation sequencing for surveys of molecular diversity of eukaryotic microorganisms. *Mol Ecol,* **19 Suppl 1,** 32-40.

<span id="_ENREF_20" class="anchor"></span>Moniz, M.B. & Kaczmarska, I. (2010) Barcoding of diatoms: nuclear encoded ITS revisited. *Protist,* **161,** 7-34.

<span id="_ENREF_21" class="anchor"></span>Moradian, M.M., Beglaryan, D., Skozylas, J.M. & Kerikorian, V. (2007) Complete Mitochondrial Genome Sequence of Three \<italic\>Tetrahymena\</italic\> Species Reveals Mutation Hot Spots and Accelerated Nonsynonymous Substitutions in \<italic\>Ymf\</italic\> Genes. *PLoS ONE,* **2,** e650.

<span id="_ENREF_22" class="anchor"></span>Nassonova, E., Smirnov, A., Fahrni, J. & Pawlowski, J. (2010) Barcoding amoebae: comparison of SSU, ITS and COI genes as tools for molecular identification of naked lobose amoebae. *Protist,* **161,** 102-115.

<span id="_ENREF_23" class="anchor"></span>Pawlowski, J., Audic, S.p., Adl, S., Bass, D., Belbahri, L.d., Berney, C.d., Bowser, S.S., Cepicka, I., Decelle, J., Dunthorn, M., Fiore-Donno, A.M., Gile, G.H., Holzmann, M., Jahn, R., Jirk‚âà√ò, M., Keeling, P.J., Kostka, M., Kudryavtsev, A., Lara, E., Luke‚âà¬∞, J., Mann, D.G., Mitchell, E.A.D., Nitsche, F., Romeralo, M., Saunders, G.W., Simpson, A.G.B., Smirnov, A.V., Spouge, J.L., Stern, R.F., Stoeck, T., Zimmermann, J., Schindel, D. & de Vargas, C. (2012) CBOL Protist Working Group: Barcoding Eukaryotic Richness beyond the Animal, Plant, and Fungal Kingdoms. *PLoS Biol,* **10,** e1001419.

<span id="_ENREF_24" class="anchor"></span>Prescott, D.M. (1994) The DNA of ciliated protozoa. *Microbiol Rev,* **58,** 233-267.

<span id="_ENREF_25" class="anchor"></span>Slapeta, J., Moreira, D. & Lopez-Garcia, P. (2005) The extent of protist diversity: insights from molecular ecology of freshwater eukaryotes. *Proc Biol Sci,* **272,** 2073-2081.

<span id="_ENREF_26" class="anchor"></span>Stern, R.F., Horak, A., Andrew, R.L., Coffroth, M.A., Andersen, R.A., Kupper, F.C., Jameson, I., Hoppenrath, M., Veron, B., Kasai, F., Brand, J., James, E.R. & Keeling, P.J. (2010) Environmental barcoding reveals massive dinoflagellate diversity in marine environments. *PLoS ONE,* **5,** e13991.

<span id="_ENREF_27" class="anchor"></span>Valentini, A., Pompanon, F. & Taberlet, P. (2009) DNA barcoding for ecologists. *Trends Ecol Evol,* **24,** 110-117.

<span id="_ENREF_28" class="anchor"></span>Walsh, P.S., Metzger, D.A. & Higuchi, R. (1991) Chelex 100 as a medium for simple extraction of DNA for PCR-based typing from forensic material. *Biotechniques,* **10,** 506-513.

<span id="_ENREF_29" class="anchor"></span>Zufall, R.A., Dimon, K.L. & Doerder, F.P. (2013) Restricted distribution and limited gene flow in the model ciliate *Tetrahymena thermophila*. *Molecular Ecology,* **22,** 1081-1091.
