Supplementary information for Altermatt et al. Methods in Ecology and Evolution. DOI: 10.1111/2041-210X.12312

**“Big answers from small worlds: a user's guide for protist microcosms as a model system in ecology and evolution” **

Altermatt F, Fronhofer EA, Garnier A, Giometto A, Hammes F, Klecka J, Legrand D, Mächler E, Massie TM, Pennekamp F, Plebani M, Pontarp M, Schtickzelle N, Thuillier V & Petchey OL

**2.6 RAMAN microspectroscopy**

**Introduction**

Raman microspectroscopy (RMS) yields information about the chemical composition of individual cells. Raman spectra result from the inelastic scattering of photons from a sample (Raman effect). That is, the scattered photons posses a wavelength/energy that is different from that of the incident light (monochromatic laser). The change in wavelength/energy during the scattering process is caused by the interaction of the photon with vibrational modes of the various chemical bonds of the molecules within a sample (e.g., C=O or C–H) ([Wagner 2009](#_ENREF_5)). Raman scattering provides detailed information about the chemical composition of a sample (molecular structure, cellular composition or, physiological state of the sample), which is summarized in the Raman spectrum ([Huang *et al.* 2010](#_ENREF_2)).

Two extensions of RMS are of special interest for experiments with microorganisms. First, a combination with stable isotope probing (SIP). Li et al. 2013 ([2013](#_ENREF_4)) demonstrated that RMS is able to detect isotopic shifts to higher wavelengths (or lower wavenumbers, wavelength<sup>-1</sup>), so called “red-shifting”, in the Raman spectra when replacing <sup>12</sup>C with <sup>13</sup>C carbon. The calculated red shift ratio (RSR) is highly correlated with the <sup>13</sup>C-content of the cells. Thus, combining SIP with RMS bears great potential for ecological experiments, such as tracking the flow of elements through food webs on a single-cell basis ([Abraham 2014](#_ENREF_1)). Moreover, using SIP with RMS is non-invasive, which stays in contrast to destructive methods such as 16S-rRNA sequencing. Second, a combination with fluorescence in-situ hybridization (FISH). Because FISH adds detailed information about the spatial structure of a cell, the combination with RMS (Raman-FISH) gives an interesting tool for single cell structure function analyses in protist populations/communities ([Huang *et al.* 2007](#_ENREF_3)).

The herein given protocol includes all necessary steps after the sampling procedure and preparations needed before analysis with RMS. This comprises the cleaning of protists and bacteria as well as the transfer to quartz slides used later for RMS, that is we cover all preparation steps specific to protists. We do not provide a protocol for the RMS analysis itself since highly specific expertise is known, such that RMS should be performed in collaboration with individuals that have the expertise and the devices to analyse samples of microorganisms.

**Materials**

***Equipment***

-   MgF<sub>2</sub> or CaF<sub>2</sub> microscope slides (© Crystran Limited).

-   (Plastic) Petri dishes (60 x 15 mm).

-   Micropipettes (10, 100, 1.000 µL).

-   Stereomicroscope (magnification 10–50 times, depending on organism size).

-   Eppendorf tubes.

***Reagents***

-   Bacterial buffer (or similar liquid) to clean protists.  This liquid should not contain any of the elements that may be part of the later analysis, such as carbon when using stable isotope probing (SIP).

**Procedure**

*Isolate and clean ciliates from culture liquid. *

This has to be done to remove influences that might potentially disturb/influence the spectra obtained from RMS. This is especially true when labelling individuals by stable isotope probing and or fixation chemicals. However, we do not recommend the use of fixation chemicals since they might influence the RMS output when being absorbed/adsorbed by a cell.  

1.  Put 3 mL of bacterial buffer in a plastic Petri dish (5 cm in diameter). 

2.  Select the protists under the stereomicroscope with a micropipette out of the sample volume and put the individuals in the Petri dish containing bacterial buffer. Take care that as little as possible is transferred from the rest of the culture to guarantee a high dilution and cleaning! E.g., if 100 µL of culture liquid are transferred together with twenty ciliates the dilution is 100 / 3.000 ≈ 3.3 %.

3.  Select the protists out of this Petri dish as described in the previous step and put them in another Petri dish containing bacterial buffer.  

4.  Repeat step 3 several times to make sure that the protists are well cleaned. The number of repetitions depends on the volume of culture liquid transferred which each ciliate. The larger the volume the more repetitions it takes to get properly cleaned protists. 

*Isolate and clean bacteria from culture liquid*

Be aware that other organisms might get destroyed during centrifugation!

1.  Take 1 mL of experimental volume and put this in an Eppendorf tube.

2.  Centrifuged this volume at 3000 rpm for 10 min. 

3.  Remove as much of the liquid phase as possible (using a micropipette) and re-suspend the residue at the bottom (bacterial pellet) with 200 µL. 

4.  Repeat steps 2 and 3 two times (or more often if desired).  

*Prepare slides for RMS *

1.  Put cleaned protists individuals or bacteria in small droplets on MgF<sub>2</sub>/CaF<sub>2</sub> slides. These slides are highly light translucent which is a prerequisite for successful application of monochrome light (laser) used in RMS.

2.  Let them dry until all liquid is vaporized.

3.  The organisms are now ready for RMS analysis.

**References**

<span id="_ENREF_1" class="anchor"></span>Abraham, W.-R. (2014) Applications and impacts of stable isotope probing for analysis of microbial interactions. *Applied Microbiology and Biotechnology,* **98,** 4817-4828.

<span id="_ENREF_2" class="anchor"></span>Huang, W.E., Li, M., Jarvis, R.M., Goodacre, R. & Banwart, S.A. (2010) Shining Light on the Microbial World: The Application of Raman Microspectroscopy. *Advances in Applied Microbiology, Vol 70* (eds A.I. Laskin, S. Sariaslani & G.M. Gadd), pp. 153-186.

<span id="_ENREF_3" class="anchor"></span>Huang, W.E., Stoecker, K., Griffiths, R., Newbold, L., Daims, H., Whiteley, A.S. & Wagner, M. (2007) Raman-FISH: combining stable-isotope Raman spectroscopy and fluorescence in situ hybridization for the single cell analysis of identity and function. *Environmental Microbiology,* **9,** 1878-1889.

<span id="_ENREF_4" class="anchor"></span>Li, M., Huang, W.E., Gibson, C.M., Fowler, P.W. & Jousset, A. (2013) Stable Isotope Probing and Raman Spectroscopy for Monitoring Carbon Flow in a Food Chain and Revealing Metabolic Pathway. *Analytical Chemistry,* **85,** 1642-1649.

<span id="_ENREF_5" class="anchor"></span>Wagner, M. (2009) Single-Cell Ecophysiology of Microbes as Revealed by Raman Microspectroscopy or Secondary Ion Mass Spectrometry Imaging. *Annu. Rev. Microbiol.,* **63,** 411–429.
